src/controllers/ildcp-host.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const Accounts = require('../services/accounts')
+const { Writer } = require('oer-utils')
+const log = require('../common').log.create('ildcp-host')
+
+class IldcpHostController {
+  constructor (deps) {
+    this.accounts = deps(Accounts)
+  }
+
+  async handle (sourceAccount, data) {
+    log.debug('responding to ILDCP config request. clientName=' + sourceAccount)
+
+    const info = this.accounts.getInfo(sourceAccount)
+
+    const writer = new Writer()
+    writer.writeVarOctetString(Buffer.from(sourceAccount, 'ascii'))
+    writer.writeUInt8(info.currencyScale)
+    writer.writeVarOctetString(Buffer.from(info.currency, 'utf8'))
+    return writer.getBuffer()
```

Comment:
This encoding should probably be added to the `ilp-packet` module

Reply:
Yeah, I wasn't sure where to put it. It's not ilp-packet layer, so I don't think ilp-packet is actually the right place unless we turn that into ilp-formats. Plus - is it really worth modularizing four lines of code?

Reply:
Fair enough